src/projections/polyfill-colspan.js

```diff
@@ -0,0 +1,87 @@
+import { last } from '../utils';
+
+function processRow({ tds = [] }, rowSpans) {
+  let start = 0;
+  const retRowSpans = rowSpans;
+  tds.forEach((td) => {
+    while (retRowSpans[start] > 0) {
+      retRowSpans[start] -= 1;
+      start += 1;
+    }
+
+    const newStart = start + (td.props.colspan || 1);
```

Comment:
We need to check and warn `colspan == 0` on non-tailing cells.

Reply:
You can put a TODO here

Reply:
Done